bridges/evm/contracts/example/DEX.sol

```diff
@@ -1,129 +1,102 @@
-            key = hex"00000003";
-        } else if (strCmp(symbol, "BTC")) {
-            key = hex"00000004";
-        } else if (strCmp(symbol, "EOS")) {
-            key = hex"00000005";
-        } else if (strCmp(symbol, "ETC")) {
-            key = hex"00000006";
-        } else if (strCmp(symbol, "ETH")) {
-            key = hex"00000007";
-        } else if (strCmp(symbol, "LTC")) {
-            key = hex"00000008";
-        } else if (strCmp(symbol, "TRX")) {
-            key = hex"00000009";
-        } else if (strCmp(symbol, "XRP")) {
-            key = hex"0000000A";
-        } else {
-            revert("UNKNOWN_SYMBOL");
-        }
-        return _balances[account][key];
-    }
-
-    function bytesToPrices(bytes memory _b)
-        public
-        pure
-        returns (uint256, uint256)
-    {
-        require(_b.length >= 16, "INVALID_LENGTH");
-        uint256 ethPrice;
-        uint256 otherPrice;
-        for (uint256 i = 0; i < 8; i++) {
-            ethPrice = ethPrice + (uint256(uint8(_b[i])) << (8 * (7 - i)));
-            otherPrice =
-                otherPrice +
-                (uint256(uint8(_b[i + 8])) << (8 * (7 - i)));
-        }
-        return (ethPrice, otherPrice);
+    return _balances[account][key];
+  }
+
+  function bytesToPrices(bytes memory _b)
+    public
+    pure
+    returns (uint256, uint256)
+  {
+    require(_b.length == 16, "INVALID_LENGTH");
+    uint256 ethPrice;
+    uint256 otherPrice;
+    for (uint256 i = 0; i < 8; i++) {
+      ethPrice = ethPrice + (uint256(uint8(_b[i])) << (8 * (7 - i)));
+      otherPrice = otherPrice + (uint256(uint8(_b[i + 8])) << (8 * (7 - i)));
     }
+    return (ethPrice, otherPrice);
+  }
 
-    function buy(bytes memory _reportPrice) public payable {
-        Bridge.VerifyOracleDataResult memory result = bridge.relayAndVerify(
-            _reportPrice
-        );
+  function buy(bytes memory _reportPrice) public payable {
+    IBridge.VerifyOracleDataResult memory result = bridge.relayAndVerify(
+      _reportPrice
+    );
 
-        require(result.codeHash == codeHash, "INVALID_CODEHASH");
+    require(result.codeHash == codeHash, "INVALID_CODEHASH");
 
-        (uint256 ethPrice, uint256 otherPrice) = bytesToPrices(result.data);
+    (uint256 ethPrice, uint256 otherPrice) = bytesToPrices(result.data);
 
-        uint256 tokenEarn = msg.value.mul(ethPrice).div(otherPrice);
+    uint256 tokenEarn = msg.value.mul(ethPrice).div(otherPrice);
 
-        _balances[msg.sender][result.params] = _balances[msg.sender][result
-            .params]
-            .add(tokenEarn);
-    }
+    _balances[msg.sender][result.params] = _balances[msg.sender][result.params].add(tokenEarn);
+  }
 
-    function sell(uint256 amount, bytes memory _reportPrice) public {
-        Bridge.VerifyOracleDataResult memory result = bridge.relayAndVerify(
-            _reportPrice
-        );
+  function sell(uint256 amount, bytes memory _reportPrice) public {
```

Comment:
Let's underscore prefix all the parameters.

Reply:
done